CHANGELOG.rst

```diff
@@ -11,6 +11,28 @@
 Changelog
 +++++++++
 
+0.14.0
+======
+
+- Obey the ``--skip-subprojects`` option passed to the ``meson
+  install`` command.
+- Implement support for building wheels targerting the Python limited
+  API. Extension modules targeting the Python limited API can be
+  easily build starting with the upcomin Meson 1.3.0 release.
+- Instead than silently using the version declared in ``meson.build``,
+  raise an error when ``pyproject.toml`` does not contain a
+  ``version`` field and ``version`` is not declared dynamic.
+- Fix the mtime of source files in the sdist tarball.
+- Fix the installation of files via ``install_subdir()``. For the
+  ``exclude_directories`` and ``exclude_files`` arguments to work,
+  Meson version 1.1.0 or later is required.
+- Add ``objc`` and ``objcpp`` compilers to the cross file generated
+  when the ``$ARCHFLAGS`` is set.
+- Extensive documentation improvements.
```

Comment:
How about adding here:

- `meson install` is no longer run as part of the wheel build process, resulting in less log being written to stdout.

Reply:
I'm not sure it is worth mentioning it, we just got rid of a single line looking something like `+ meson install --quiet --no-rebuild --allow-dirty`

Reply:
maybe not indeed. okay, resolving.